app/gui/qt/mainwindow.h

```diff
@@ -49,7 +49,6 @@ class SonicPiUDPServer;
 
 struct help_page {
   QString title;
-  QString keyword;
```

Comment:
fixed with https://github.com/hzulla/sonic-pi/commit/65e4ff9718de0d73c408cfc597f09c060b2ec25e - keyword is back.